main.css

```diff
@@ -0,0 +1,91 @@
+/* fuentes externas*/
+@font-face {
+  font-family: "NotoSerif";
+  src: url(/assets/tipografia/NotoSerif-Regular.ttf);
+  font-display: swap;
+}
```

Comment:
Please add a line break so it aligns with the code style.

Reply:
thanks for the observation, already working on it